scripts/plots/cam/exevs_cam_radar_plots.sh

```
@@ -178,10 +178,12 @@ fi
 ###################################################################
 # Copy Plots Output to Main Directory
 ###################################################################
+shopt -s nullglob
```

@MarcelCaron-NOAA What does this line (and shopt) do? Do you know if this is a function that you can use in operations?

CC @SamiraArdani-NOAA @PerryShafran-NOAA

---

For "ls", when a wildcard pattern doesn’t match any files, there are two possible bash behaviors, and the default behavior is to leave the pattern unchanged (e.g., ${DATA}/out/workdirs/*), and we get the "No such file ...." warning.

shopt -s nullglob enables the other behavior, where unmatched patterns expand to null, and that's what I want here, since it's expected that the files might be missing on a restart.  If the pattern expands to null, the following code block would just be skipped.  I revert to the default behavior after this code block so that other code is not affected.

I didn’t see anything in the [NCO Implementation Standards](https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf) that prohibits shopt, and since this isn’t suppressing errors, just controlling glob behavior, NCO might even prefer it.. That said, let me know if it raises any flags and I’ll consider alternatives!